js/content/store.js

```diff
@@ -1385,10 +1385,24 @@ class AppPageClass extends StorePageClass {
         if (this.isDlc()) { return; }
 
         let appid = this.appid;
-        let response = await Background.action("appdetails", {"appids": appid, "filters": "support_info"});
-        if (!response || !response[appid] || !response[appid].success) { return; }
+        let cache = LocalStorage.get("support_infos", {});
+        for (let id in cache) {
+            if (!cache[id].expiry || cache[id].expiry < Math.trunc(Date.now() / 1000)) {
```

Comment:
```suggestion
            if (cache[id].expiry < Math.trunc(Date.now() / 1000)) {
```
Why do the check here?

Reply:
To check if the `expiry` key even exists.

Reply:
I know, but why shouldn't it?

Reply:
A bug perhaps?